src/BloomExe/ToPalaso/BetterToolTip.cs

```diff
@@ -22,40 +22,41 @@ namespace Bloom.ToPalaso
 	/// EnhancedToolTip supports the ToolTipWhenDisabled and SizeOfToolTipWhenDisabled
```

Comment:
This comment should be updated to talk about BetterToolTip. Probably just a rename that got missed, but if this comment is really about some other class we need one about THIS class.